lms/djangoapps/course_wiki/plugins/markdownedx/mdx_mathjax.py

```diff
@@ -4,9 +4,7 @@
 import markdown
 
 try:
```

Comment:
Let's remove this try/except block altogether.

Reply:
I tried removing it but it gives python failure

Reply:
What's the error?

Reply:
Cannot import etree from markdown